charts/nodejs/templates/scale-pods-right-to-api.yaml

```diff
@@ -0,0 +1,23 @@
+{{ if and (contains "api" .Values.biomageCirepo) (or (eq .Values.kubernetes.env "staging") (eq .Values.myAccount.accountId "160782110667")) }}
```

Comment:
why restricting it to account id `160782110667`?

Reply:
The intention here is minimal permissions. It would only be enabled in staging and production HMS. If Biomage intends to use this for production environments as well then the additional check could be removed or we could base it on a configurable variable.